src/main/java/FrameworksDrivers/CompareGraphics.java

```diff
@@ -0,0 +1,136 @@
+package FrameworksDrivers;
+
+import Entities.Person;
+import Entities.Schedule;
+import Entities.events.Event;
+import InterfaceAdapters.ScheduleManager;
+
+import javax.swing.*;
+import java.awt.*;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import javax.swing.JFrame;
+import java.util.Objects;
+
+public class CompareGraphics extends JFrame implements ActionListener {
+    private Graphics g2g;
+    private ScheduleManager manager;
+    private Person user1;
+    private Person user2;
+
+    private JLabel titleLabel;
+
+    private String[] timesList = {"00:00", "01:00", "02:00", "03:00", "04:00", "05:00", "06:00", "07:00", "08:00", "09:00", "10:00", "11:00", "12:00", "13:00", "14:00", "15:00", "16:00", "17:00", "18:00", "19:00", "20:00", "21:00", "22:00", "23:00"};
+
+
+    public CompareGraphics(Person user1, Person user2, ScheduleManager manager) {
+        super("Schedule Compare");
+        this.user1 = user1;
+        this.user2 = user2;
+        setSize(850, 850); // setting the default size of the frame
+
+        setLayout(null);
+        titleLabel = new JLabel("Comparing Schedule");
+        titleLabel.setFont(new Font("Verdana", Font.PLAIN, 18)); // setting different font for title
+        titleLabel.setBounds(360, -5, 200, 50);
+        add(titleLabel);
+
+        setLocationRelativeTo(null);
+        setVisible(true);
+
+    }
+
+    public void paint(Graphics g) {
+        super.paint(g);
+
+        g2g = g;
+
+        // creating custom colors
+        Color color0 = Color.white;
+        Color color1 = new Color(110, 205, 250);
+        Color color2 = new Color(39, 208, 245);
+        Color color3 = new Color(0, 210, 231);
+        Color color4 = new Color(0, 211, 207);
+        Color color5 = new Color(0, 210, 175);
+        Color color6 = new Color(0, 207, 137);
+        Color color7 = new Color(65, 202, 94);
+
+        Color[] colorsList = {color0, color1, color2, color3, color4, color5, color6, color7};
+
+        String[] titles = {"Time", "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday", "Sunday"};
```

Comment:
this is the same as titles on line 102 at the bottom -- could it be made into an instance attribute or something to avoid repeated code?